chemprop/cli/utils/parsing.py

```diff
@@ -213,17 +216,39 @@ def build_data_from_files(
     smiss, rxnss, Y, weights, lt_mask, gt_mask = parse_csv(
         p_data, smiles_cols, rxn_cols, target_cols, ignore_cols, weight_col, bounded, no_header_row
     )
-    X_f = np.load(p_features) if p_features else None
-    V_fs = np.load(p_atom_feats, allow_pickle=True) if p_atom_feats else None
-    E_fs = np.load(p_bond_feats, allow_pickle=True) if p_bond_feats else None
-    V_ds = np.load(p_atom_descs, allow_pickle=True) if p_atom_descs else None
+    n_molecules = len(list(zip(*smiss)))
+
+    X_ds = load_input_features(p_features, n_molecules, feature="X_d")
```

Comment:
```suggestion
    X_ds = load_input_features(p_descs, n_molecules, feature="X_d")
```
Perhaps rename p_features. If you do, it should be changed in the signature of the function as well.